frontend/src/components/pages/teacher/ClassroomsPage.tsx

```diff
@@ -136,83 +67,95 @@ const ClassroomsPage = (): React.ReactElement => {
+                      paddingBottom="6"
+                      paddingTop="6"
+                      spacing="6"
+                      width="100%"
+                    >
+                      {totalPages > 1 && (
+                        <Pagination
+                          currentPage={currentPage}
+                          onPageChange={setCurrentPage}
+                          pagesCount={totalPages}
                         />
-                      </GridItem>
-                    ))}
-                  </Grid>
-                  <VStack
-                    alignItems="center"
-                    paddingBottom="6"
-                    paddingTop="6"
-                    spacing="6"
-                    width="100%"
-                  >
-                    {totalPages > 1 && (
-                      <Pagination
-                        currentPage={currentPage}
-                        onPageChange={setCurrentPage}
-                        pagesCount={totalPages}
-                      />
-                    )}
-                  </VStack>
-                </TabPanel>
-                <TabPanel padding="0">
-                  <h1>Coming soon!</h1>
-                </TabPanel>
-              </TabPanels>
-            </Tabs>
-          </>
-        ) : (
-          <>
-            <Center
-              backgroundColor="blue.50"
-              borderRadius="1rem"
-              color="blue.300"
-              minWidth="100%"
-              pb={14}
-            >
-              <MessageContainer
-                buttonText="Add New Classroom"
-                image={DisplayAssessmentsIllustration}
-                onClick={handleAddClassroom}
-                paragraphs={[
-                  "Click on the below button to create your first classroom",
-                ]}
-                subtitle="You currently have no classrooom."
-                textColor="blue.300"
-              />
-            </Center>
-          </>
-        )}
-      </Box>
+                      )}
+                    </VStack>
+                  </TabPanel>
+                  <TabPanel padding="0">
+                    <h1>Coming soon!</h1>
+                  </TabPanel>
+                </TabPanels>
+              </Tabs>
+            </>
+          ) : (
+            <>
+              <Center
+                backgroundColor="blue.50"
+                borderRadius="1rem"
+                color="blue.300"
+                minWidth="100%"
+                pb={14}
+              >
+                <MessageContainer
+                  buttonText="Add New Classroom"
+                  image={DisplayAssessmentsIllustration}
+                  onClick={handleAddClassroom}
+                  paragraphs={[
+                    "Click on the below button to create your first classroom",
+                  ]}
+                  subtitle="You currently have no classrooom."
+                  textColor="blue.300"
+                />
+              </Center>
```

Comment:
nit: can we put this in a separate container to follow the existing pattern & to keep this page a bit leaner? similar to `ErrorState`, `TestSubmissionMessage`, `EmptyTableState`, etc.